src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.ts

```diff
@@ -136,7 +136,15 @@ export class PoolFormComponent implements OnInit {
           validators: [Validators.required, Validators.min(1)]
         }),
         ecOverwrites: new FormControl(false),
-        compression: compressionForm
+        compression: compressionForm,
+        max_bytes: new FormControl(0, {
+          validators: [Validators.required],
```

Comment:
By reading the error messages I think you are missing the `min` validator:
```suggestion
          validators: [Validators.required, Validators.min(0)],
```

Reply:
removed required error condition and allow empty field.